runtime/jcl/common/mgmtmemory.c

```diff
@@ -702,9 +703,72 @@ Java_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificati
 				return;
 			}
 
-			(*env)->CallVoidMethod(env, threadInstance, helperGCID,
-					gcName, gcAction, gcCause,
-					(jlong)gcInfo->index, (jlong)gcInfo->startTime, (jlong)gcInfo->endTime,
+			/* Error tracepoint for the cases
+			 * the value of init or max is negative but not -1; or
+			 * used is greater than the value of committed; or
+			 * committed is greater than the value of max if max is not -1.
+			 */
+			if (TrcEnabled_Trc_JCL_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificationLoop_memoryUsageError
+				|| TrcEnabled_Trc_JCL_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificationLoop_memoryUsage
+				) {
+				const char *nsGcName = (*env)->GetStringUTFChars(env, gcName, NULL);
+				for (idx = 0; idx < gcInfo->arraySize; ++idx) {
+					U_64 preUsed = gcInfo->preUsed[idx];
+					U_64 preCommitted = gcInfo->preCommitted[idx];
+					I_64 preMax = gcInfo->preMax[idx];
+					U_64 postUsed = gcInfo->postUsed[idx];
+					U_64 postCommitted = gcInfo->postCommitted[idx];
+					I_64 postMax = gcInfo->postMax[idx];
+					I_64 initialSize = gcInfo->initialSize[idx];
+					const char *nsPoolName = (mgmt->supportedMemoryPools > idx)
+							? mgmt->memoryPools[idx].name
+							: mgmt->nonHeapMemoryPools[idx - mgmt->supportedMemoryPools].name;
+
+					if ((preUsed > preCommitted)
+						|| ((-1 < preMax) && (preCommitted > preMax))
```

Comment:
This does not match the comment on line 707; this was (and should continue to be) `-1 != preMax`.
Same comment about `postMax` on line 731.